sensor/AdblockPlugin.js

```diff
@@ -209,6 +217,8 @@ class AdblockPlugin extends Sensor {
 
     async updateFilter() {
       const config = await this.getAdblockConfig();
+      const diffList = this.lastOnList.filter(x => !this.currentOnList.includes(x))
```

Comment:
it's not lastOnList vs. currentOnList. It's user config vs. cloud default config

1. if there is any user config, use user config
2. if any list in the user config is not available on the cloud any more, remove this list from effective list
3. if there is no user config, use cloud default config